examples/cryptocurrency-advanced/backend/src/proto/cryptocurrency.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum.examples.cryptocurrency_advanced;
+
+import "helpers.proto";
+
+message Transfer {
+ exonum.PublicKey to = 1;
```

Comment:
Weird indentation (1). Is there canonical indentation for `proto` files, btw? The [examples](https://developers.google.com/protocol-buffers/docs/proto3) seem to use 2-space indentation.

Reply:
clang-format can be used for this.